doc/flame/effects.md

```diff
@@ -111,6 +111,7 @@ offset is relative to the current position of the target:
 ```
 
 ```dart
+
```

Comment:
why empty line at the start of every code block?

Reply:
I tried to resolve the merge conflict and it added this.

Reply:
fixed now